### ISSUE TITLE
LA bag ingest - Rita Guibert Collection resource with no component id

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/figgy#3016]
One item in the Rita Guibert Collection of Latin American Authors came through with no title, presumably because it couldn't find a component id.

https://figgy-staging.princeton.edu/catalog/6704289f-1a1b-4e35-a13d-4e7cf81e70a5

@kellybolding

**Comment by kellybolding** [comment]
@hackmastera For each of these three collections that you just made issues for, it looks like one resource was created for any loose files that weren't matched to a component ID. Because they don't all go with the same component ID, I addressed each of the problem audio files within these resources and made notes in a spreadsheet here --there's a sheet per collection (https://docs.google.com/spreadsheets/d/1L74AvaernYHErzNgmtvvczRtVDSIrjNknaUc-kWXbRQ). I didn't make any notes for the XML or jpg files because I'm assuming those will get associated with the correct component once the EADs are corrected to reflect the names of the audio files.

For this collection, I just committed some corrections to the EAD, which were mainly typos. For two files (32101047381569_1_p1 and 32101047381569_1_p2), these file names were correct in the EAD, but it looks like the ingest process may have missed them because they don't exactly follow the expected naming convention. There is an extra _p1 and _p2 at the end because the vendor had to break up the recording for that side into two parts. This also happens for the errors with C0652, so perhaps we can either modify the matching process to account for this variation and/or ingest these manually? (The component IDs they should be associated with are listed in the spreadsheet.)
